ledger/daml-on-sql/src/main/scala/on/sql/Main.scala

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.on.sql
+
+import com.daml.ledger.participant.state.v1
+import com.daml.ledger.participant.state.v1.SeedService.Seeding
+import com.daml.platform.configuration.{InvalidConfigException, LedgerConfiguration}
+import com.daml.platform.sandbox.cli.Cli
+import com.daml.platform.sandbox.config.{LedgerName, SandboxConfig}
+import com.daml.platform.sandbox.{GlobalLogLevel, SandboxServer}
+import com.daml.resources.ProgramResource
+
+object Main {
+
+  private val Name = LedgerName("DAML-on-SQL")
+
+  private[sql] val defaultConfig: SandboxConfig =
+    SandboxConfig.defaultConfig.copy(
+      participantId = v1.ParticipantId.assertFromString("daml-on-sql-participant"),
```

Comment:
Maybe `-participant` is redundant? In the logs it should anyway be logged as key-value pair.

Reply:
Happy to change it to simply `"daml-on-sql"`.

Reply:
Done.